SDKs/PowerShell/JumpCloud.SDK.DirectoryInsights/JumpCloud.SDK.DirectoryInsights.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Version>0.0.33</Version>
+    <Version>0.0.34</Version>
```

Comment:
Looks like we're updating DI version here for some reason

Reply:
These are previous changes from Geoffs branch, not releasing in this PR